module-services/service-desktop/ServiceDesktop.cpp

```diff
@@ -51,6 +51,36 @@ auto ServiceDesktop::getSerialNumber() const -> std::string
     return settings->getValue(std::string("factory_data/serial"), settings::SettingsScope::Global);
 }
 
+auto ServiceDesktop::generateDeviceUniqueId() -> void
+{
+    const auto deviceUniqueId = utils::generateRandomId(sdesktop::DeviceUniqueIdLength);
```

Comment:
I believe this note was not tackled in other PR where I wrote it: why do we have some self-grown rand generation when we have TRNG on the chip? we use then this data for cache ciphering so I would assume the better the random number the better? :)

Reply:
It's a good suggestion and should be tracked under separate a task.

Reply:
and this is the second time my suggestion is closed without me knowing :) if it will be tracked, can you please at least link issue here?